ViAn/GUI/mainwindow.ui

```diff
@@ -17,35 +17,44 @@
    <double>1.000000000000000</double>
   </property>
   <widget class="QWidget" name="centralWidget">
-   <layout class="QVBoxLayout" name="verticalLayout_2">
+   <layout class="QVBoxLayout" name="verticalLayout">
```

Comment:
This layout is for the whole centralwidget (including the project view, video player, and bookmark view) and within this layout there is a splitter with horisontal orientation. So your proposed name would be incorrect here.